packages/events/src/router.ts

```diff
@@ -53,15 +69,35 @@ export const appRouter = router({
       return getEventById(input)
     }),
     actions: router({
-      create: publicProcedure
+      notify: publicProcedure
+        .input(
+          z.object({
+            eventId: z.string(),
+            transactionId: z.string(),
+            action: NotifyActionInput
+          })
+        )
+        .mutation(async (options) => {
+          return addAction(options.input.eventId, {
+            ...options.input.action,
+            type: 'NOTIFY',
```

Comment:
Using these through the `ActionType.NOTIFY` will make changing them much easier in the future.